src/bindings/js/node/src/preprocess/pre_post_process_wrap.cpp

```diff
@@ -54,6 +55,26 @@ Napi::Value PrePostProcessorWrap::input(const Napi::CallbackInfo& info) {
     return obj;
 }
 
+Napi::Value PrePostProcessorWrap::output(const Napi::CallbackInfo& info) {
+    if (info.Length() != 0 && info.Length() != 1) {
+        reportError(info.Env(), "Wrong number of parameters.");
+        return info.Env().Undefined();
```

Comment:
Why is there a return after `reportError` in one place, but not in another?

Reply:
discussed offline